include/NAS2D/Trig.h

```diff
@@ -23,6 +23,8 @@ extern const float RAD2DEG;
 float degToRad(float degree);
 float radToDeg(float rad);
 float angleFromPoints(float x, float y, float x2, float y2);
+Vector<float> angleToDirection(float angle);
+[[deprecated("Please use angleToDirection instead")]]
```

Comment:
As much as I'd like to approve this, I can't. Microsoft has decided that using the `deprecated` attribute should always result in an error, not a warning as the specification says it should.

Consider changing it to `#pragma message` to get around the problem.

Reply:
I think that depends on the warning level and having warnings as errors enabled.

I suppose I could just remove the old function. As far as I'm aware, it's only used one place in the test code, so should be easy enough to update the call while updating the NAS2D submodule. It was probably overly conservative of me to keep the old method and deprecate it.

Reply:
> I think that depends on the warning level and having warnings as errors enabled.

Nope. Regardless of warning level and if WAE is on or off, still emits an error. Moot now, since you just removed the old function.